### PR TITLE
Issue 717 push for label changes

### PR DESCRIPTION
**Pull request by vttranlina**
resolve https://github.com/linagora/tmail-backend/issues/717

### CODE REVIEW

File: tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/label/LabelChangeRepository.scala [hunk 1]
```diff
@@ -50,6 +52,17 @@ case class LabelChange(accountId: AccountId,
   override def getAccountId: AccountId = accountId
 }
 
+case object LabelTypeName extends TypeName {
+  override val asString: String = "Label"
```
Comment (quantranhong1999) [inline tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/label/LabelChangeRepository.scala:56]: Maybe add a line example in `pushWithFirebase.adoc`?

**Review by quantranhong1999 — COMMENTED** [review]
read it, nothing to add

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Rebase needed ;-)

**Comment by vttranlina** [comment]
rebase

**Review by quantranhong1999 — APPROVED** [review]

**Pull request merged**